source/opcode.cpp

```diff
@@ -719,3 +719,14 @@ std::vector<uint32_t> spvOpcodeMemorySemanticsOperandIndices(SpvOp opcode) {
       return {};
   }
 }
+
+bool spvOpcodeIsAccessChain(SpvOp opcode) {
+  switch (opcode) {
+    case SpvOpAccessChain:
+    case SpvOpInBoundsAccessChain:
+    case SpvOpPtrAccessChain:
```

Comment:
Why don't you include "OpInBoundsPtrAccessChain"?

Reply:
I just missed it because it was not in sequence with the others in the specification.
Done.